catroid/src/main/java/org/catrobat/catroid/ui/fragment/ProjectListFragment.java

```diff
@@ -369,24 +372,23 @@ public boolean onOptionsItemSelected(MenuItem item) {
 	}
 
 	private void initializeDefaultProjectAfterDelete() {
-		final ProjectManager projectManager = ProjectManager.getInstance();
 		getActivity().findViewById(R.id.fragment_container).setVisibility(View.GONE);
 		getActivity().findViewById(R.id.progress_circle).setVisibility(View.VISIBLE);
```

Comment:
IMO the code is easier to understand when using onPreExecute() for this code.
* pre (Show loading spinner)
* run (init default project and update list)
* post (Hide loading spinner)

What do you think?